methcomp/blandaltman.py

```diff
@@ -3,34 +3,65 @@
 import matplotlib.transforms as transforms
 import numpy as np
 from scipy import stats
-from typing import Tuple
+from typing import Tuple, Dict, Any
 
 __all__ = ["blandaltman"]
 
 
-class _BlandAltman(object):
-    """Internal class for drawing a Bland-Altman plot"""
+class BlandAltman:
+    """Class for drawing a Bland-Altman plot"""
+
+    DEFAULT_POINTS_KWS = {"s": 20, "alpha": 0.6, "color": "#000000"}
+
+    def __init__(
+        self, method1, method2, diff: str, limit_of_agreement: float, CI: float
+    ):
+        """
+
+        Parameters
+        ----------
+        diff : "absolute"  or "percentage"
+        The difference to display, whether it is an absolute one or a percentual one.
+        If None is provided, it defaults to absolute.
+        limit_of_agreement : float, optional
+        Multiples of the standard deviation to plot the limit of afgreement bounds at.
+        This defaults to 1.96.
+        CI : float, optional
+        The confidence interval employed in the mean difference and limit of agreement
+        lines. Defaults to 0.95.
+        """
 
-    def __init__(self, method1, method2,
-                 diff, limit_of_agreement, CI):
         # variables assignment
+        self.computed = False
         self.method1: np.array = np.asarray(method1)
-        self.method2: np.array = np.asarray(method2) 
+        self.method2: np.array = np.asarray(method2)
         self.diff_method: str = diff
         self.CI: float = CI
         self.loa: float = limit_of_agreement
 
         # check provided parameters
         self._check_params()
 
-    def compute(self) -> dict:
-        # perform necessary calculations and processing
+    def compute(self) -> Dict[str, Any]:
+        """Calculates the statistics for method comparison using
+        Bland-Altman plotting. Returns a dictionary with the results.
+
+        Parameters
```

Comment:
I think Parameters is not needed if there are no docs, not sure how the documentation site works though

Reply:
Would prefer to keep it for now, currently documentation script needs it.